packages/synapse-react-client/src/components/SynapseTable/SendToCavaticaConfirmationDialog.tsx

```diff
@@ -108,6 +109,8 @@ export default function SendToCavaticaConfirmationDialog(
     },
   )
 
+  const allCompleteAndIncompleteActions = useTrackTransientListItems(actions)
```

Comment:
Track actions for Send to Cavatica case

Reply:
This works for me, assuming the confirmation button enable state is still based on the actual actions rather than the transient completed and incomplete action list